### DISCUSSION TITLE
zoomImg Props

### DISCUSSION THREAD

**Discussion by lucasartsfr** [rpearce/react-medium-image-zoom#394] [Q&A] [answered]
Hello,

Thanks for this Amazing Lib ! I tried to change image on Zoom with the props : zoomImg, but it does not work (I use NextJS 13).

Tried many things : with a new Image() or  something like this : 

```
<Zoom  zoomImg={{src: {Img.src}}} >
            
                    <NextImage  
                    layout="responsive"
                    id={id} 
                    width={Img.Ratio.Width/Factor} 
                    height={Img.Ratio.Height/Factor} 
                    alt={Img.Nom} 
                    src={Img.Path}
                />
</Zoom>
```

 etc.. but can't find a way to make it Works. Can you help me please ? Maybe I am doing something wrong
 
 EDIT : Ok. For Next We need to set : srcset, not just src and it's working. Sorry for inconvenience

**Comment by rpearce** [comment, marked as answer]
@lucasartsfr No worries! If you have any further issues or questions and can provide an example through CodeSandbox or a repository, I can help debug it when I get time.
